tempesta_fw/http.c

```diff
@@ -3887,6 +4044,48 @@ static TfwCfgSpec tfw_http_specs[] = {
 		.allow_none = true,
 		.cleanup = tfw_cfgop_cleanup_whitelist_mark,
 	},
+	{
+		.name = "http_uri_brange",
```

Comment:
Just a notice, no fix is required. This class of directives is not about http processing, it's more about parsing and string operations. But I can't say that moving the directives  to `http_parser.c` or `str_simd.c` will make huge difference.